docs/reference/analysis/tokenfilters/multiplexer-tokenfilter.asciidoc

```diff
@@ -0,0 +1,114 @@
+[[analysis-multiplexer-tokenfilter]]
+=== Multiplexer Token Filter
+
+A token filter of type `multiplexer` will emit multiple tokens at the same position,
+each version of the token having been run through a different filter.
+
+Note that the child filters will in effect be passed a mock tokenstream consisting
```

Comment:
I might put this as a `WARNING: ` in the `Options` section. And I'd flip the sentence around to something like "Shingle or multi-word synonym token filters will not function normally when they are declared in the `filters` array because they read ahead internally which is unsupported by the `multiplexer`.